src/jlapi.c

```diff
@@ -352,6 +352,30 @@ JL_DLLEXPORT void (jl_cpu_wake)(void)
     jl_cpu_wake();
 }
 
+#ifndef _OS_WINDOWS_
+#  define UNW_LOCAL_ONLY
+#  include <libunwind.h>
+#define STRINGIFY1(x) #x
+#define STRINGIFY2(x) STRINGIFY1(x)
+asm("_jl_unw_getcontext: movq " STRINGIFY2(unw_tdep_getcontext) "@GOTPCREL(%rip), %rax\n"
```

Comment:
Yeah, I think I'll do that for now. We can revisit this for the other architectures later and see what would be more natural in context.